src/components/RegisterForm/RegisterForm.jsx

```diff
@@ -77,100 +93,128 @@ const RegisterForm = () => {
-            touched: formik.touched.userName,
-          }}
-        />
+    <Background>
+      <RegisterFormContainer>
+        <SignInFormStyles onSubmit={formik.handleSubmit} autoComplete="off">
+          <SignInTitle>Create account</SignInTitle>
+          <SignInText>
+            Already have an account? <LoginLink>Log in</LoginLink>
+          </SignInText>
+          <ItemWrapp>
+            <StyledLabel
+              color={{
+                error: formik.errors.userEmail,
+                touched: formik.touched.userEmail,
+              }}
+              htmlFor="email"
+            />
+            <StyledInput
+              id="userEmail"
+              name="userEmail"
+              type="email"
+              onChange={formik.handleChange}
+              value={formik.values.userEmail}
+              placeholder="E-mail"
+              color={{
+                error: formik.errors.userEmail,
+                touched: formik.touched.userEmail,
+              }}
+            />
 
-        {formik.errors.userName ? (
-          <Error>{formik.errors.userName}</Error>
-        ) : !formik.errors.userName && formik.touched.userName ? (
-          <Success>Field is not empty</Success>
-        ) : null}
-      </ItemWrapp>
+            {formik.errors.userEmail ? (
+              <Error>{formik.errors.userEmail}</Error>
+            ) : !formik.errors.userEmail && formik.touched.userEmail ? (
+              <Success>Field is not empty</Success>
+            ) : null}
+          </ItemWrapp>
+          <ItemWrapp>
+            <StyledLabel
+              color={{
+                error: formik.errors.userName,
+                touched: formik.touched.userName,
+              }}
+              htmlFor="Name"
+            />
+            <StyledInput
+              id="userName"
+              name="userName"
+              type="text"
+              onChange={formik.handleChange}
+              value={formik.values.userName}
+              placeholder="Username"
+              color={{
+                error: formik.errors.userName,
+                touched: formik.touched.userName,
+              }}
+            />
+            {formik.errors.userName ? (
+              <Error>{formik.errors.userName}</Error>
+            ) : !formik.errors.userName && formik.touched.userName ? (
+              <Success>Field is not empty</Success>
+            ) : null}
+          </ItemWrapp>
+          <ItemWrapp>
+            <StyledLabel
+              color={{
+                error: formik.errors.password,
+                touched: formik.touched.password,
+              }}
+              htmlFor="password"
+            />
+            <StyledInput
+              id="password"
+              name="password"
+              type="password"
+              onChange={formik.handleChange}
+              value={formik.values.password}
+              placeholder="Password"
+              color={{
+                error: formik.errors.password,
```

Comment:
You can check if input has any errors and has been touched in this component and pass appropriate prop to the styled component so that you could avoid havy logic inside.